Acumatica.RESTClient/BaseApi/ODataAPI.cs

```diff
@@ -1,23 +1,58 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Text;
-
 using Acumatica.RESTClient.Client;
-
 using RestSharp;
+using RestSharp.Authenticators;
 
 namespace Acumatica.RESTClient.Api
 {
+   
     public class ODataAPI : BaseApi
```

Comment:
If we use "RestRequest" for sending the API call, does it make sense to have it derived from BaseApi?